docs/noobaa.md

```diff
@@ -0,0 +1,45 @@
+# Noobaa Commands
+
+The Noobaa command supports all the noobaa cli arguments. See [noobaa cli](https://github.com/noobaa/noobaa-operator?tab=readme-ov-file#noobaa-cli)
```

Comment:
Would the odf cli tool really support all the noobaa cli arguments? For example, do `install` and `upgrade` make sense in the cli tool? Shouldn't those be through standard ODF install/upgrade?

Reply:
Good question, I'm checking with the build team regarding the commands supports,.

Reply:
Actually, there were few changes in upstream vs downstream commands in noobaa. See below output